.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,28 @@
+Check all that apply
+
+# Type of Change
+
+- [ ] WWST Certification Request
+     - If this is your first time contributing code:
```

Comment:
Something that was brought up reviewing the template I was proposing for our internal repos was that using checkboxes for mutually exclusive choices causes GitHub to render the "tasks" progress as if it has incomplete elements.

I received two different proposals on how to address that:

1. Reword the checkbox items to indicate that picking that box means that you've done one of the possible options, or
2. GitHub actually supports multiple PR templates (I didn't know that at the time I started the internal one), and you could have different templates for different PR "types" instead of indicating the type with a checkbox choice.

Granted, none of the above is a dealbreaker. But I think it's a good goal to shoot for having the checkbox configuration be set up in such a way that a properly processed PR template results in 100% of "tasks completed".

Reply:
back in my day we had a name for mutually-exclusive checkboxes and they were called radio buttons 😠

Reply:
I think option 1 that Doug mentioned should at least be tried to make the WWST/1st contribution section not be so out of place on some PRs. We are the ones doing the most PRs on the repo, and that section doesn't apply to any of those changes.